modules/Navbar.js

```diff
@@ -0,0 +1,51 @@
+import { DateTime } from '../node_modules/luxon/build/es6/luxon.js';
+
+const formSection = document.getElementById('form-section');
+const formLink = document.getElementById('form-link');
+const homeLink = document.getElementById('home-link');
+const contactLink = document.getElementById('contact-link');
+const booksSection = document.getElementById('all-books');
+const titleSection = document.getElementById('titleSection');
+const contactSection = document.getElementById('contact-section');
+
+formSection.style.display = 'none';
+contactSection.style.display = 'none';
+
+formLink.addEventListener('click', () => {
+  formSection.style.display = 'flex';
+  booksSection.style.display = 'none';
+  titleSection.style.display = 'none';
+  contactSection.style.display = 'none';
+  formLink.classList.add('active');
+  homeLink.classList.remove('active');
+  contactLink.classList.remove('active');
+});
+
+homeLink.addEventListener('click', () => {
+  formSection.style.display = 'none';
+  booksSection.style.display = 'block';
+  contactSection.style.display = 'none';
+  titleSection.style.display = 'block';
+  homeLink.classList.add('active');
+  formLink.classList.remove('active');
+  contactLink.classList.remove('active');
+});
+
+contactLink.addEventListener('click', () => {
+  formSection.style.display = 'none';
+  booksSection.style.display = 'none';
+  titleSection.style.display = 'none';
+  contactSection.style.display = 'block';
+  contactLink.classList.add('active');
+  homeLink.classList.remove('active');
+  formLink.classList.remove('active');
+});
+
+const dateSection = document.getElementById('dateLine');
+const displayCurrentDate = () => {
+  const date = DateTime.now();
+  const formattedDate = date.toLocaleString(DateTime.DATETIME_MED_WITH_SECONDS);
+  dateSection.innerHTML = formattedDate;
+};
```

Comment:
- [ ] the date is not displayed please make sure to fix it so that it is shown as expected 
<img width="1432" alt="Screenshot 2023-03-27 at 15 22 23" src="https://user-images.githubusercontent.com/61790587/227986834-7a55156c-9e63-42db-a55f-4061f8ff0a24.png">